docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -336,9 +341,12 @@ a detection rule exception (`detection`) or an endpoint exception (`endpoint`).
 |==============================================
 |Name |Type |Description
 
-|threat_filter |Object[]
+|threat_filters |Object[]
 |{ref}/query-filter-context.html[Query and filter context] array used to filter
 documents from the {es} index containing the threat values.
+
+|threat_indicator_path |String
+|Much like an ingest processor, users can use this field to define where their threat indicator can be found on their indicator documents.
```

Comment:
@spong , @rylnd looks like the wording for threat_indicator_path is:

```
Much like an ingest processor, users can use this field to define where their threat indicator can be found on their indicator documents.
```

in the docs and on the UI it is:

```
Specify the document path containing your threat indicator fields. Used for enrichment of indicator match alerts. Defaults to threat.indicator unless otherwise specified.
```

Do you want it to be more like the UI for consistency and clarity?

```
Specifies the document path containing your threat indicator fields. Used for enrichment of indicator match alerts. Defaults to threat.indicator unless otherwise specified.
```

Reply:
IMO more variations is better for something like this: if a user still has questions after reading these docs, they may gain a better understanding of the concept after reading the field description in the rule form, or finding relevant commits/comments in kibana if they're particularly tech-savvy.

Note also that [the default value, and the field description](https://github.com/elastic/kibana/pull/92081) both changed for 7.12. I think it'd be useful for 7.12 docs to mention that the default here is `threatintel.indicator`.